schemas/rum/view-schema.json

```diff
@@ -313,6 +313,11 @@
               "description": "Whether this session is currently active. Set to false to manually stop a session",
               "default": true,
               "readOnly": true
+            },
+            "sampled_for_replay": {
+              "type": "boolean",
```

Comment:
🥜 (nitpick) I'm fine with this or either an `"enum": [ true ]` if this value will always be true when sent

Reply:
I had in mind to also send `false` values, it seems helpful to have `true` and `false` values in the events.
Any reason to not send them?

Reply:
It was either that model, or the `has_replay` one. Works either way!